meta-resin-common/recipes-support/resin-hostname/resin-hostname/resin-hostname

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -e
+
+. /usr/sbin/resin-vars
+
+if [ -z "$NEW_HOSTNAME" ]; then
+    NEW_HOSTNAME=$(jq -r '.uuid //empty' $CONFIG_PATH)
+fi
+
+if [ -z "$NEW_HOSTNAME" ]; then
+    echo "[ERROR]: Unexpected uuid missing from config.json."
+    exit 1
+fi
+
+# Set hostname temporary hostname
+hostnamectl set-hostname "$NEW_HOSTNAME"
```

Comment:
I will address this after we merge this PR, so when I test the ro PR this service will fail.